Cargo.toml

```diff
@@ -8,7 +8,7 @@ name = "weggli_native"
 crate-type = ["cdylib", "staticlib"]
 
 [dependencies]
-libc = "0.2"
+libc = "0.2.127"
```

Comment:
Do we need this exact pin?

Reply:
@woodruffw, I was still testing the changes. It was giving me libc version error and minimum required was 0.2.127. But it seems due to xcode update and command line tools were not set correctly. I rolled back the changes.